.github/workflows/upload-and-deploy.yaml

```diff
@@ -233,7 +233,18 @@ jobs:
       - name: "Deploy sceptre stacks to dev"
         run: pipenv run sceptre --var "namespace=${{ env.NAMESPACE }}" launch develop --yes
 
-      - name: Configure S3 to Glue lambda with S3 trigger
+      - name: Delete preexisting S3 event notification for this namespace
```

Comment:
Does anything need to be done in the cleanup workflow? https://github.com/Sage-Bionetworks/recover/blob/058b0779c7692055eecd1e6bd1e6fd3bfe7aa704/.github/workflows/cleanup.yaml#L4

Reply:
Good idea. We will delete the namespace's event configuration from the input bucket.